posthog/api/test/test_authentication.py

```diff
@@ -440,3 +442,83 @@ def test_e2e_test_special_handlers(self):
                 "/api/reset/e2e_test_user/", {"token": "e2e_test_token", "password": "a12345678"}
             )
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+
+class TestPersonalAPIKeyAuthentication(APIBaseTest):
+    def test_personal_api_key_updates_last_used_at_hourly(self):
```

Comment:
non-blocking

given you've got another test for this with the year different, is there a case for adding one for the month and day also? I may have misunderstood the reason for the within_the_year test though.

Reply:
was just testing the two extremes, but is a bit void now with the timedelta